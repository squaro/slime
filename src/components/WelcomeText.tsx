import styled from 'styled-components';

const WelcomeTextParagraph = styled.p`
  text-align: center;
  color: #000000;
`;

const WelcomeText = (): JSX.Element => (
  <WelcomeTextParagraph>
    Press <b>Start</b> to begin tracking
    <br />
    the round's direction using
    <br />
    an interactive arrow!
  </WelcomeTextParagraph>
);

export default WelcomeText;
