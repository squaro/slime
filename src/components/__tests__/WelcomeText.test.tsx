import { render } from '@testing-library/react';
import WelcomeText from '../WelcomeText';

describe('WelcomeText', (): void => {
  it('should load welcome text component', (): void => {
    // Act
    render(<WelcomeText />);

    // Assert
    expect(document.body).toMatchSnapshot();
  });
});
