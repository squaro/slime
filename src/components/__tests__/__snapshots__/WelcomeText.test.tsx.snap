// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeText should load welcome text 1`] = `
<body>
  <div>
    <p
      class="sc-bdfBwQ cwXRm"
    >
      Press 
      <b>
        Start
      </b>
       to begin tracking
      <br />
      the round's direction using
      <br />
      an interactive arrow!
    </p>
  </div>
</body>
`;
